### PR TITLE
docs: document `HOMEBREW_INSTALL_FROM_API` usage in installation

### PR DESCRIPTION
**Pull request by kidonng**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

This feature is added in https://github.com/Homebrew/install/pull/628, but I forgot to document it here.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -25,6 +25,17 @@ export HOMEBREW_CORE_GIT_REMOTE="..."  # put your Git mirror of Homebrew/homebre
 
 The default Git remote will be used if the corresponding environment variable is unset.
 
+## Skip Tap Cloning
```
Comment (MikeMcQuaid) [inline docs/Installation.md:28]:
```suggestion
## Skip Tap Cloning (beta)
```

File: docs/Installation.md [hunk 2]
```diff
@@ -25,6 +25,17 @@ export HOMEBREW_CORE_GIT_REMOTE="..."  # put your Git mirror of Homebrew/homebre
...
+
+You can instruct Homebrew to skip cloning the Homebrew/homebrew-core tap during installation by setting `HOMEBREW_INSTALL_FROM_API` in your shell environment with this script:
```
Comment (MikeMcQuaid) [inline docs/Installation.md:30]:
```suggestion
You can instruct Homebrew to skip cloning the Homebrew/homebrew-core tap during installation by setting the beta `HOMEBREW_INSTALL_FROM_API` environment variable with the following:
```

**Review by MikeMcQuaid — COMMENTED** [review]
Looks good! Some safety tweaks.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @kidonng!

**Review by Rylan12 — COMMENTED** [review]
It's probably worth clarifying that you also need to set `HOMEBREW_INSTALL_FROM_API` in your shell profile file so it persists in new sessions. Otherwise, the next time you try to `brew install` in a new session, it will try to tap homebrew/core again then.

**Comment by kidonng** [comment]
The "Git Remote Mirroring" section above does not emphasize this point, I'm just following suite. But [the install script does mention setting	the remotes](https://github.com/Homebrew/install/blob/309e9d089e9d51d8f20254c30a7ce8d4607cf8c1/install.sh#L1032-L1034), perhaps this should resolved there.

**Pull request merged**